apps/tsconfig.json

```diff
@@ -26,6 +26,7 @@
     "paths": {
       "@cdo/apps/*": ["./src/*"],
       "@cdo/static/*": ["./static/*"],
+      "@cdo/i18n/*": ["./i18n/*"],
```

Comment:
@code-dot-org/platform specifically wanted signoff on this path alias. To clarify, this only affects code at compile-time and won't actually replace any paths at build-time.